src/charm.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+# Copyright 2024 mark
```

Comment:
Copyright should be to Canonical

Reply:
Oups - auto generated by charmcraft init.